### ISSUE TITLE
Multiple features for forecasting

### ISSUE THREAD

**Issue by gidim** [facebookincubator/prophet#181]
Is it possible to use multiple features to predict y_hat ? Basically combining regression and forecasting.

**Comment by zad0xlik** [comment]
I have a very similar question and did not want to open up a new issue. I am trying to recreate a forecast using Prophet and compare to what I have already done in SAS. 

In addition to past values of the response series and past errors, I want to also model the response series using the current and past values of other series, called input series.

For example, in SAS this process would look like:

List the input series in a CROSSCORR= option on the IDENTIFY statement and specify how they enter the model with an INPUT= option on the ESTIMATE statement:

proc arima data=input_data_set;
   identify var=sales crosscorr=price;
   estimate input=price;
run;

In the code above, series called PRICE helps model SALES. This example performs a simple linear regression of SALES on PRICE; it produces the same results as PROC REG or another SAS regression procedure.

Can this process (adding input variables) be done using Prophet? Please point to an example if possible.

**Comment by bletham** [comment]
This is something that fits very easily into the modeling framework for Prophet but hasn't yet been implemented. It is on the roadmap in #101 and so hopefully will be coming soon!

**Comment by axerocks** [comment]
I was wondering if this feature has been incorporated.

**Comment by sammourad** [comment]
@axerocks yep, it's [here](https://facebook.github.io/prophet/docs/seasonality,_holiday_effects,_and_regressors.html#additional-regressors)
